marlowe-cli/src/Language/Marlowe/CLI/Command/Util.hs

```diff
@@ -278,12 +281,14 @@ faucetCommand network socket =
-    <*> (O.option $ Lovelace <$> O.auto)   (O.long "lovelace"      <> O.metavar "LOVELACE"    <> O.value 1_000_000_000 <> O.help "The lovelace to send to each address."                                                                           )
-    <*> O.strOption                        (O.long "out-file"      <> O.metavar "FILE"                                 <> O.help "Output file for transaction body."                                                                               )
-    <*> (O.optional . O.option O.auto)     (O.long "submit"        <> O.metavar "SECONDS"                              <> O.help "Also submit the transaction, and wait for confirmation."                                                         )
-    <*> O.some (O.argument parseAddressAny $                          O.metavar "ADDRESS"                              <> O.help "The addresses to receive the funds."                                                                             )
+    <$> O.option parseNetworkId            (O.long "testnet-magic"   <> O.metavar "INTEGER"     <> network               <> O.help "Network magic. Defaults to the CARDANO_TESTNET_MAGIC environment variable's value."                              )
+    <*> O.strOption                        (O.long "socket-path"     <> O.metavar "SOCKET_FILE" <> socket                <> O.help "Location of the cardano-node socket file. Defaults to the CARDANO_NODE_SOCKET_PATH environment variable's value.")
+    <*> (O.option $ Lovelace <$> O.auto)   (O.long "lovelace"        <> O.metavar "LOVELACE"    <> O.value 1_000_000_000 <> O.help "The lovelace to send to each address."                                                                           )
+    <*> O.strOption                        (O.long "out-file"        <> O.metavar "FILE"                                 <> O.help "Output file for transaction body."                                                                               )
+    <*> (O.optional . O.option O.auto)     (O.long "submit"          <> O.metavar "SECONDS"                              <> O.help "Also submit the transaction, and wait for confirmation."                                                         )
+    <*> O.option parseAddressAny           (O.long "faucet-address"  <> O.metavar "ADDRESS"                              <> O.help "The faucet addresses to provide funds."                                                                          )
+    <*> O.strOption                        (O.long "required-signer" <> O.metavar "SIGNING_FILE"                         <> O.help "File containing a required signing key."                                                                         )
+    <*> O.some (O.argument parseAddressAny $                            O.metavar "ADDRESS"                              <> O.help "The addresses to receive the funds."                                                                             )
```

Comment:
Horizontal/tabulated alignment causes needlessly noisy diffs ;)

Reply:
Yes, maybe we can enforce formatting tool to replace this approach? @bwbush do you have anything against changing the formatting across the project?

Reply:
Normally I'm not that hung up about formatting choices - but horizontal alignment (though I know it's super common in Haskell) causes a specific problem in that it significantly increases the likelihood of merge conflicts that are not easy to resolve (diff and merge algorithms don't handle horizontal differences nearly as well as vertical ones)

Reply:
I'm not a fan of this horiz alignment with semigroup things either but not sure what to do. The column got wider and it looked weird to not move them all. In my own stuff I usualy opt for using more vertical space which would make these files crazy long and perhaps harder to see the "big picture" of an individual command's switches on one screen height.
Maybe keep one-line but no columns, just one space between everything.

Also, I tend to not bother qualified importing the `Options.Applicative` in my code because the `O.` is very noisy. Basically just "giving up" on the explicit imports because most of these options parsing modules are using functions from that library anyway.

Reply:
To be clear @dino-iog I'm not requesting a change here. I think if we wanted to take a position on this it should be uniformly applied across the codebase as @paluh suggested.